components/BackButton/index.test.jsx

```diff
@@ -0,0 +1,12 @@
+import { render, screen } from '@testing-library/react';
+import BackButton, { backButtonTestIds } from '.';
+
+describe('BackButton', () => {
+  it('renders a back button', () => {
+    render(<BackButton />);
+
+    const backButton = screen.getByTestId(backButtonTestIds.button);
+
+    expect(backButton).toBeVisible();
+  });
+});
```

Comment:
Should we add a test to cover the onClick action (as the action is already implemented?) —  this might be interesting as I might require to mock the next router object :)

Reply:
sure, done in 62702641a4f14b18d1a77bfdc6c11f87694fc841